src/osm/GeometryHandler.cpp

```diff
@@ -205,216 +480,221 @@ void osm2rdf::osm::GeometryHandler<W>::closeExternalStorage() {
 
 // ____________________________________________________________________________
 template <typename W>
-void osm2rdf::osm::GeometryHandler<W>::calculateRelations() {
+void GeometryHandler<W>::calculateRelations() {
   // Ensure functions can open external storage for reading.
   closeExternalStorage();
-  if (_config.writeGeometricRelationStatistics) {
-    _statistics.open();
-  }
   prepareRTree();
   prepareDAG();
   dumpNamedAreaRelations();
   dumpUnnamedAreaRelations();
   const auto& nodeData = dumpNodeRelations();
   dumpWayRelations(nodeData);
-
-  if (_config.writeGeometricRelationStatistics) {
-    std::cerr << std::endl;
-    std::cerr << osm2rdf::util::currentTimeFormatted()
-              << " Closing statistics files..." << std::endl;
-    _statistics.close("[\n", "{}\n]");
-    std::cerr << osm2rdf::util::currentTimeFormatted() << " ... done"
-              << std::endl;
-  }
 }
 
 // ____________________________________________________________________________
 template <typename W>
-void osm2rdf::osm::GeometryHandler<W>::prepareRTree() {
-  std::cerr << osm2rdf::util::currentTimeFormatted()
-            << " Packing area r-tree with " << _spatialStorageArea.size()
-            << " entries ... " << std::endl;
-  _spatialIndex =
-      SpatialIndex(_spatialStorageArea.begin(), _spatialStorageArea.end());
-  std::cerr << osm2rdf::util::currentTimeFormatted() << " ... done"
-            << std::endl;
+void GeometryHandler<W>::prepareRTree() {
+  // empty the rtree!
```

Comment:
This comment does not correspond to any line. Nearest would be 518 which replaces the index with a new one.

Reply:
518 moved to 526 but this is still relevant as the rtree is never emptied but replaced during `_spatialIndex = SpatialIndex(values.begin(), values.end());`